modules/lvms-upgrading-lvms-on-sno.adoc

```diff
@@ -3,32 +3,34 @@
 // storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="lvms-upgrading-lvms-on-sno_{context}"]
-= Upgrading {lvms} on {sno} clusters
+[id="lvms-updating-lvms_{context}"]
+= Updating {lvms} on a single-node OpenShift cluster
```

Comment:
```suggestion
= Updating {lvms} on a {sno} cluster
```

Reply:
I am not sure who mentioned this during one of the peer reviews - adding attributes in Titles is not suggested as it might cause issues while rendering the final output.

@jeana-redhat WDYT?

Reply:
It should work in a module, but will not in an assembly. This one already has `{lvms}` so it seems to be working

Reply:
Sure! Thanks for the clarification!
Updated as per the suggestion.